### DISCUSSION TITLE
Unstyled PrimeVue still includes styles in bundle?

### DISCUSSION THREAD

**Discussion by AcidRaZor** [primefaces/community#787] [PrimeVue] [answered]
Hi,

I just started using PrimeVue as a proof of concept together with Tailwind. I've followed the instructions carefully.

```
import Button from 'primevue/button'
import Breadcrumb from 'primevue/breadcrumb'

app.use(PrimeVue, { unstyled: true })

app.component('Button', Button)
app.component('Breadcrumb', Breadcrumb)
```

According to Rollup's visualizer, it still includes styles/icon references:

![image](https://github.com/primefaces/community/assets/12648519/fd8b3864-b473-428b-897c-d358784be460)

I don't mind it, but as I expand usage and replacing my components from the old component library the company was using with this, I wouldn't necessarily want to carry more and more unused weight in my chunk. For example, adding DataTable, Column, ColumnGroup and Row, this grows the chunk by about 300kb (pre-gzip)

So my question is, did I miss something in my configuration? Is this expected? Is there a recommended manual chunking method for prime (if yes, how?)

I'm quite new to Vue 3 + Vite and Prime, my previous experience only includes writing libraries in TypeScript. So apologies in advance if I'm being a dumbass :)

**Comment by tugcekucukoglu** [comment, marked as answer]
It's on the roadmap. We're planning to work for.

Thank you!
